README.md

```diff
@@ -362,7 +365,7 @@ defp deps do
     {:gettext, "~> 0.11"},
     {:jason, "~> 1.0"},
     {:plug_cowboy, "~> 2.0"},
-    {:phoenix_live_view, "~> 0.8.1"},
+    {:phoenix_live_view, "~> 0.9.0"},
```

Comment:
The explanation above needs also to be changed:

```
Append the following line to the end of the list:
```
{:phoenix_live_view, "~> 0.9.0"},
```

Reply:
The references to the code need also to be updated to link the lines of code to this PR:

> 🏁 [`mix.exs`](https://github.com/dwyl/phoenix-liveview-counter-tutorial/blob/d0521ced8cb9d55a6a214a886ecab811ee065960/mix.exs#L33-L46)
file at the end of Step 2:
[mix.exs#L44](https://github.com/dwyl/phoenix-liveview-counter-tutorial/blob/d0521ced8cb9d55a6a214a886ecab811ee065960/mix.exs#L44)

Reply:
@SimonLab good catch! Thanks for updating the link. 👍

Reply:
This should now be resolved: https://github.com/dwyl/phoenix-liveview-counter-tutorial/pull/6/commits/58a105ced960a078dc911c64157827f9290acc73